build.sbt

```diff
@@ -32,7 +32,7 @@ val pomInfo = (
 
 lazy val commonSettings = Seq(
   organization := "com.intenthq.pucket",
-  version := "1.5.0",
+  version := "1.5.1",
```

Comment:
I think we'll have to go to `1.6.0` here as it's not strictly binary compatible because we've moved some methods into the base classes and added some defaults for a few params (which is fine, not a biggie at all, but semver major needed).